e2e/cases/server/custom-server/scripts/pureServer.mjs

```diff
@@ -48,9 +48,6 @@ export async function startDevServerPure(fixtures) {
     await rsbuildServer.afterListen();
   });
 
-  // subscribe the server's http upgrade event to handle WebSocket upgrade
-  server.on('upgrade', rsbuildServer.onHTTPUpgrade);
```

Comment:
should we call `rsbuildServer.connectSocket()` instead?

Reply:
The `connectSocket` and `onHTTPUpgrade` methods are not needed in this case because compilation is not triggered here.